app/assets/stylesheets/application.scss

```diff
@@ -217,6 +217,67 @@ body {
   display: none;
 }
 
+.pagination {
+  background: #edeeea;
+  cursor: default;
+  /* self-clearing method: */
+}
+.pagination a,
+.pagination span,
+.pagination em {
+  padding: 0.2em 0.5em;
+  display: block;
+  float: left;
+  margin-right: 1px;
+}
+.pagination .disabled {
+  color: #999999;
+  border: 1px solid #dddddd;
+}
+.pagination .current {
+  font-style: normal;
+  font-weight: bold;
+  background: #2e6ab1;
+  color: white;
+  border: 1px solid #2e6ab1;
+}
+.pagination a {
+  text-decoration: none;
+  color: #105cb6;
+  border: 1px solid #9aafe5;
+}
+.pagination a:hover,
+.pagination a:focus {
+  color: #000033;
+  border-color: #000033;
+}
+.pagination .page_info {
+  background: #2e6ab1;
+  color: white;
+  padding: 0.4em 0.6em;
+  width: 22em;
+  margin-bottom: 0.3em;
+  text-align: center;
+}
+.pagination .page_info b {
+  color: #000033;
+  background: #6aa6ed;
+  padding: 0.1em 0.25em;
+}
+.pagination:after {
+  content: ".";
+  display: block;
+  height: 0;
+  clear: both;
+  visibility: hidden;
+}
+* html .pagination {
+  height: 1%;
+}
+*:first-child + html .pagination {
+  overflow: hidden;
+}
```

Comment:
I got all this CSS  as a recommendation from the `wil_paginate` gem documentation.

Reply:
oo nice. Shout out documentation!

Reply:
looking good